sc/social/like/subscribers.py

```diff
@@ -107,3 +113,40 @@ def assign_canonical_url(obj, event):
             'Canonical domain not set in Social Media configlet; '
             "Facebook's Open Graph canonical URL (og:orl) will not be available"
         )
+
+
+def social_content_check(obj, event):
+    """Check if content is valid if enabled in sc.social.content, in action when edit."""
+    request = getRequest()
+
+    # FIXME: view still available with package uninstalled; why?
```

Comment:
Removing this code will cause the `ComponentLookupError` error when installing a Plone site with  plone.app.contenttypes